lib/pbench/server/database/models/datasets.py

```diff
@@ -194,16 +183,16 @@ class MetadataKeyError(MetadataError):
     It is never raised directly, but may be used in "except" clauses.
     """
 
+    pass
+
 
 class MetadataMissingParameter(MetadataKeyError):
     """A Metadata required parameter was not specified."""
 
     def __init__(self, what: str):
+        super().__init__(f"Metadata must specify a {self.what}")
         self.what = what
```

Comment:
At line 193, `self.what` doesn't have a value yet; presumably it should be just `what`.

Reply:
I caught most of those (I thought) when I moved the f-strings, but I'm not entirely shocked (albeit a bit disappointed) that I missed a few.